### PR TITLE
stolostron/backlog#20969 hyper endpts 25

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: install/requirements.adoc [hunk 1]
```diff
@@ -197,6 +197,20 @@ When you are installing bare metal managed clusters with the Hive Operator, whic
...
+=== Hosted control planes networking requirements (Technology Preview)
+
+When you use hosted control planes, the `HypershiftDeployement` resource must have connectivity to the following endpoints:
+
+* {ocp-short} master and worker nodes
+* AWS API
+* S3 API
+* Azure API
+* Registration agents for all managed clusters
+* {ocp-short} image repositories that store the ISO images of the coreOS and teh image registry for {ocp-short} pods
```
Comment (jnpacker) [inline install/requirements.adoc]: typo. `teh`

Comment (jnpacker) [inline install/requirements.adoc]: It's the Red Hat image repositories. Its not all ISO's there are container images too.

File: install/requirements.adoc [hunk 2]
```diff
@@ -197,6 +197,20 @@ When you are installing bare metal managed clusters with the Hive Operator, whic
...
+* {product-title-short} hub connects to the klusterlet that runs on a hosted cluster
```
Comment (jnpacker) [inline install/requirements.adoc]: hub connects to the klusterlet (on per hosted cluster) that runs on the hosting cluster.

File: install/requirements.adoc (hunk 1 above)
Comment (cadawson) [reply install/requirements.adoc]: I think I addressed this one, @jnpacker .

File: install/requirements.adoc (hunk 2 above)
Comment (cadawson) [reply install/requirements.adoc]: Updated the wording on this one.

**Pull request closed without merging**